### DISCUSSION TITLE
Human readable date in logs

### DISCUSSION THREAD

**Discussion by jaime-pineiro-imatia** [gotenberg/gotenberg#862] [Q&A] [answered]
Date of log events is shown in timestamp shape which is not human friendly to know when the event has occurred. Is there a way to configure logs to show event date in a human readable format (Something like "2024-04-02 11:10:11.443")?

**Comment by gulien** [comment]
If a terminal is attached (and the log format = `json`), you should see the following:

```
{"level":"info","ts":"2024/04/23 14:23:44.577","logger":"api","msg":"request handled","trace":"c2963532-ea20-4fbf-b7b1-915c8fe09b17","remote_ip":"192.168.65.1","host":"localhost:2000","uri":"/health","method":"GET","path":"/health","referer":"","user_agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36","status":200,"latency":307582,"latency_human":"307.582µs","bytes_in":0,"bytes_out":173}
```

For production usage, most logging tools (datadog for instance) use timestamp AFAIK.

**Reply by jaime-pineiro-imatia** [reply]
Thanks @gulien, i just tried that but i still see date in ts format

> $ docker run -p 3000:3000 gotenberg/gotenberg:8.4 gotenberg --log-level=info --log-format=json
> 
>   _____     __           __
>  / ___/__  / /____ ___  / /  ___ _______ _
> / (_ / _ \/ __/ -_) _ \/ _ \/ -_) __/ _ '/
> \___/\___/\__/\__/_//_/_.__/\__/_/  \_, /
>                                    /___/
> 
> A Docker-powered stateless API for PDF files.
> Version: 8.4.0
> -------------------------------------------------------
> [SYSTEM] modules: api chromium exiftool libreoffice libreoffice-api libreoffice-pdfengine logging pdfcpu pdfengines pdftk prometheus qpdf webhook
> [SYSTEM] libreoffice-api: LibreOffice ready to start
> [SYSTEM] prometheus: collecting metrics
> [SYSTEM] chromium: Chromium ready to start
> [SYSTEM] pdfengines: exiftool libreoffice-pdfengine pdfcpu pdftk qpdf
> [SYSTEM] api: server listening on port 3000
> {"level":"info","ts":1713882822.6193187,"logger":"api","msg":"request handled","trace":"e8b8d0f0-487b-47d8-af27-4d840e6e32c8","remote_ip":"172.17.0.1","host":"localhost:3000","uri":"/forms/chromium/convert/url","method":"POST","path":"/forms/chromium/convert/url","referer":"","user_agent":"curl/7.68.0","status":200,"latency":1516928783,"latency_human":"1.516928783s","bytes_in":196,"bytes_out":95600}

Is there something that i'm missing?

**Reply by gulien** [reply, marked as answer]
Try `docker run -it -p3000:3000 …`
